src/WebJobs.Extensions.ApiHub/Table/TableEntityValueBinder.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
```

Comment:
All of your new files need file headers. Aren't you getting build warnings/errors for this? You should be.